src/Options.php

```diff
@@ -848,6 +848,9 @@ public function getDpi()
      */
     public function setFontCache($fontCache)
     {
+        if (!is_dir($fontCache) && !mkdir($fontCache)) {
```

Comment:
Should we make the directory recursively (e.g., `mkdir($dir, 0777, true)`)?

Reply:
Yep, that would be definitely better, even if the 777 permission might be problematic somehow